app/src/main/java/com/example/joshuarodriguez/mvpcalculator/mvp/utils/RxBus/Observers/CalculatorOperatorButtonPressedObserver.java

```diff
@@ -0,0 +1,17 @@
+package com.example.joshuarodriguez.mvpcalculator.mvp.utils.RxBus.Observers;
+
+public abstract class CalculatorOperatorButtonPressedObserver extends BusObserver<CalculatorOperatorButtonPressedObserver.OperatorButtonValue> {
+
+    public CalculatorOperatorButtonPressedObserver() {
+        super(OperatorButtonValue.class);
+    }
+
+    public static class OperatorButtonValue {
+
```

Comment:
Got it, thanks! Please let me know if there is anything else that needs to be modified.